### PR TITLE
running multiple simulations

### PR DESCRIPTION
**Pull request by mjanczykowski**

### CODE REVIEW

File: scripts/zeus.sh [hunk 1]
```diff
@@ -1,34 +1,52 @@
...
-    -Dakka.remote.netty.tcp.hostname=${SUPERVISOR_HOSTNAME} \
-    -Dtrafficsimulation.time.seconds=20 \
-    -Dtrafficsimulation.city.cols=4 \
-    -Dtrafficsimulation.city.rows=4 \
-    -Dworker.nodes=24 \
-    -Dworker.cores=1 \
-    -Dakka.remote.log-remote-lifecycle-events=off \
-    -jar ${PBS_O_WORKDIR}/supervisor.jar &
-SUPERVISOR_PID=$!
+declare -a PARAMS_LIST
```
Comment (salceson) [inline scripts/zeus.sh:11]: Nice one, didn't know that it can be done this way :)

File: scripts/zeus.sh [hunk 2]
```diff
@@ -1,34 +1,52 @@
 #!/bin/bash
 #PBS -q l_short
-#PBS -l nodes=25:ppn=12
+#PBS -l nodes=64:ppn=12
```
Comment (salceson) [inline scripts/zeus.sh]: Have you tried running this on Zeus?

**Review by salceson — APPROVED** [review]
Good job ;)

**Comment by mjanczykowski** [comment]
My job was stuck on queue.

**Pull request merged**